crates/sui-indexer/src/indexer_reader.rs

```diff
@@ -1637,6 +1637,31 @@ impl IndexerReader {
                 )))?;
         Ok(TreasuryCap::try_from(treasury_cap_obj_object)?.total_supply)
     }
+
+    pub fn get_consistent_read_range(&self) -> Result<(i64, i64), IndexerError> {
+        let latest_checkpoint_sequence = self
+            .run_query(|conn| {
+                checkpoints::table
+                    .select(checkpoints::sequence_number)
+                    .order(checkpoints::sequence_number.desc())
+                    .first::<i64>(conn)
```

Comment:
n00b q: can we use `max` here? i.e. `select(max(checkpoints::sequence_number)).first::<i64>(conn)`. Same for the query below.